contrib/systemd/mako.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Lightweight Wayland notification daemon
+Documentation=man:mako(1)
+PartOf=graphical-session.target
```

Comment:
```suggestion
PartOf=graphical-session.target
After=graphical-session.target
```

Reply:
Should not really be needed generally, but no hard in including it I guess.